pkg/network/node/networkpolicy.go

```diff
@@ -118,6 +122,8 @@ func NewNetworkPolicyPlugin() osdnPolicy {
 		warnedPolicies:  make(map[ktypes.UID]string),
 		skippedPolicies: make(map[ktypes.UID]string),
 		localPodIPs:     make(map[ktypes.UID]string),
+
+		nsToBeCreated: make(map[uint32]interface{}),
```

Comment:
you should use `sets.Int32`. (Yeah, we're working with `uint32`s, but we know they're only 24 bits max, so they can be safely cast to `int32`.)

also, `nsToBeCreated` isn't a very descriptive name... it needs some combination of a better name, and a good comment explaining how/when/why it's used

Reply:
oh! yes, vnid is 24 bits max as per vxlan header field and using `sets.Int32` now. changed field name into `unProcessedVNIDs` with some more description. is that fine now ?